### DISCUSSION TITLE
How can i use Two Serializers in ListCreateAPIView

### DISCUSSION THREAD

**Discussion by shahensaifullah** [encode/django-rest-framework#8803] [Question & Answer]
In my API, I want to use two serializers in ListCreateAPIView. How can i use that? One is for Get method and another POST method.

**Comment by auvipy** [comment]
you can not use 2 serializer at one View. you should check https://www.django-rest-framework.org/api-guide/relations/#nested-relationships

**Reply by shahensaifullah** [reply]
I need two serializers for two request action.

**Comment by jerinpetergeorge** [comment]
It is not possible out of the box. But, if you elaborate on your use case, may be we can suggest some workarounds if any @shahriar350

**Reply by shahensaifullah** [reply]
```
def get_serializer(self):
      if self. request.method == 'GET':
           return ONESerializer
      else:
           return AnotherSerializer
```

Can i use like that?

**Reply by jerinpetergeorge** [reply]
Yes, this is possible.
